components/charts/average.vue

```diff
@@ -0,0 +1,56 @@
+<template>
+  <div class="average">
+    <div
+      :class="averageClass"
+      :style="{ width: `${average}%` }"
+    />
+  </div>
+</template>
+
+<script>
+export default {
+  props: {
+    average: {
+      type: Number,
+      required: true,
+    },
+  },
+  computed: {
+    averageClass() {
+      switch (true) {
+        case (this.average < 25):
+          return 'low';
+        case (this.average >= 25 && this.average <= 75):
+          return 'medium';
+        case (this.average > 75):
+          return 'high';
+        default:
+          return '';
+      }
```

Comment:
switch(true) seems to be fun but, it's strictly useless here.
```suggestion
      if (this.average < 25) {
        return 'low';
      } else if (this.average <= 75) {
        return 'medium';
      } else if (this.average > 75) {
        return 'high';
      }
      return '';
```